commons/Common.ml

```diff
@@ -446,7 +446,7 @@ let profile_code2 category f =
 (*****************************************************************************)
 
 let get_value filename =
-  let chan = open_in filename in
+  let chan = open_in_bin filename in
   let x = input_value chan in (* <=> Marshal.from_channel  *)
   (close_in chan; x)
```

Comment:
This would have failed (segfault) on Windows.

Reply:
Nice